### PR TITLE
Updated SDK (localfeed) and recordings

### PR DESCRIPTION
**Pull request by EvgenyAgafonchikov**
## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [ ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by maddieclayton** [comment]
@EvgenyAgafonchikov Please let me know when you are ready for one of us to take a look at this.

**Comment by MikhailTryakhov** [comment]
@MiYanni  can you please escalate this PR merging? it contains recordings really needed for contribution process

**Comment by EvgenyAgafonchikov** [comment]
@maddieclayton, please review. Currently I have to skip one test as we have no related feature activated however I hope we could activate and re-record it soon

File: src/Network/Network/AzureFirewall/ApplicationRuleCollection/NewAzureFirewallApplicationRuleCollectionCommand.cs [hunk 1]
```diff
@@ -49,7 +49,6 @@ public class NewAzureFirewallApplicationRuleCollectionCommand : NetworkBaseCmdle
         [ValidateSet(
             MNM.AzureFirewallRCActionType.Allow,
             MNM.AzureFirewallRCActionType.Deny,
-            MNM.AzureFirewallRCActionType.Alert,
```
Comment (MiYanni) [inline src/Network/Network/AzureFirewall/ApplicationRuleCollection/NewAzureFirewallApplicationRuleCollectionCommand.cs]: This is a breaking change. `Alert` was added in [Network 1.6.0](https://github.com/Azure/azure-powershell/blob/master/src/Network/Network/ChangeLog.md#version-160), and cannot be removed outside of a breaking change release. Please revert.

File: src/Network/Network/AzureFirewall/NetworkRuleCollection/NewAzureFirewallNetworkRuleCollectionCommand.cs [hunk 2]
```diff
@@ -50,7 +50,6 @@ public class NewAzureFirewallNetworkRuleCollectionCommand : NetworkBaseCmdlet
         [ValidateSet(
             MNM.AzureFirewallRCActionType.Allow,
             MNM.AzureFirewallRCActionType.Deny,
-            MNM.AzureFirewallRCActionType.Alert,
```
Comment (MiYanni) [inline src/Network/Network/AzureFirewall/NetworkRuleCollection/NewAzureFirewallNetworkRuleCollectionCommand.cs]: This is a breaking change. `Alert` was added in [Network 1.6.0](https://github.com/Azure/azure-powershell/blob/master/src/Network/Network/ChangeLog.md#version-160), and cannot be removed outside of a breaking change release. Please revert.

File: tools/StaticAnalysis/Exceptions/Az.Network/BreakingChangeIssues.csv [hunk 3]
```diff
@@ -441,3 +441,5 @@
...
 "Microsoft.Azure.PowerShell.Cmdlets.Network.dll","Microsoft.Azure.Commands.Network.SetAzureApplicationGatewayUrlPathMapConfigCommand","Set-AzApplicationGatewayUrlPathMapConfig","0","1050","The parameter set 'SetByResource' for cmdlet 'Set-AzApplicationGatewayUrlPathMapConfig' has been removed.","Add parameter set 'SetByResource' back to cmdlet 'Set-AzApplicationGatewayUrlPathMapConfig'."
+"Microsoft.Azure.PowerShell.Cmdlets.Network.dll","Microsoft.Azure.Commands.Network.NewAzureFirewallApplicationRuleCollectionCommand","New-AzFirewallApplicationRuleCollection","0","2040","The validation set for parameter 'ActionType' for cmdlet 'New-AzFirewallApplicationRuleCollection' no longer contains the value 'Alert'.","Add 'Alert' back to the validation set for 'ActionType'."
+"Microsoft.Azure.PowerShell.Cmdlets.Network.dll","Microsoft.Azure.Commands.Network.NewAzureFirewallNetworkRuleCollectionCommand","New-AzFirewallNetworkRuleCollection","0","2040","The validation set for parameter 'ActionType' for cmdlet 'New-AzFirewallNetworkRuleCollection' no longer contains the value 'Alert'.","Add 'Alert' back to the validation set for 'ActionType'."
```
Comment (MiYanni) [inline tools/StaticAnalysis/Exceptions/Az.Network/BreakingChangeIssues.csv]: After `Alert` is readded, please revert these suppressions.

File: src/Network/Network/AzureFirewall/NetworkRuleCollection/NewAzureFirewallNetworkRuleCollectionCommand.cs (hunk 2 above)
Comment (MikhailTryakhov) [reply src/Network/Network/AzureFirewall/NetworkRuleCollection/NewAzureFirewallNetworkRuleCollectionCommand.cs]: @fanymanea  please review comment from PS team. please discuss you change and revert if it's needed

**Pull request merged**